.ci/install_cc_env_ubuntu.sh

```diff
@@ -0,0 +1,69 @@
+# Copyright (c) 2017 Intel Corporation
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+#!/bin/bash
+
+function check_err {
+	if [ $? -ne 0 ]
+	then
+		exit 1
+	fi
+}
+
+echo "Add clear containers sources to apt list"
+sudo sh -c "echo 'deb http://download.opensuse.org/repositories/home:/clearlinux:/preview:/clear-containers-2.1/xUbuntu_16.10/ /' >> /etc/apt/sources.list.d/cc-oci-runtime.list"
+check_err
+
+echo "Update apt repositories"
+sudo apt-get update
```

Comment:
Not essential but you might want to add `-qq` to this call to discard output. Alternatively, you could use `chronic` as we do here:

https://github.com/01org/cc-oci-runtime/blob/master/.ci/ci-configure.sh#L44..L46